db/seeds.rb

```diff
@@ -5,3 +5,38 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
```

Comment:
Delete unnecessary comments that are automatically generated with rails.

Reply:
Will do :)